numpy/_core/fromnumeric.py

```diff
@@ -3692,6 +3698,14 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
     if mean is not np._NoValue:
         kwargs['mean'] = mean
 
+    if correction != 0:
+        if ddof != 0:
+            raise ValueError(
+                "ddof and correction can't be provided simultaneously."
+            )
+        else:
+            ddof = correction
+
     if type(a) is not mu.ndarray:
         try:
             std = a.std
```

Comment:
Good reminder: any class that defines `.std()` gets these arguments passed on. So, there is a choice to be made here: do we pass on `correction` (and not `ddof`) if it is passed in, or do we, as implemented now, just set `ddof=correction`. I think the choice here is fine, but raising the possible issue just in case (I have had to implement `.std()` in some astropy classes, but the choice here would not prevent me from adding `correction`; I'd still need to support `ddof` anyway...).

Reply:
Thank you for pointing it out. I also think that the current setup is fine: Array API defines `std` and `var` only in the main namespace. There's no `std` and `var` for the Array Object, so there's no need to introduce `correction` to `numpy.ndarray` class.